__snapshots__/yoshi-go.js

```diff
@@ -0,0 +1,34 @@
+exports['CHANGES-go-yoshi-message'] = `
+chore: created CHANGES.md
+`
+
+exports['CHANGES-go-yoshi'] = `
+# Changelog
+
+## [0.124.0](https://www.github.com/googleapis/yoshi-go-test-repo/compare/v0.123.4...v0.124.0) 
+
+
+### Features
+
+* **all:** auto-regenerate gapics ([#1000](https://www.github.com/googleapis/yoshi-go-test-repo/issues/1000)) ([#1001](https://www.github.com/googleapis/yoshi-go-test-repo/issues/1001))
```

Comment:
we're okay with having the module name in **bold** rather than the current formatting of your changelog?

Reply:
I am fine either way. Bolding is fine.